components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/main/java/org/wso2/carbon/identity/cors/mgt/core/constant/SchemaConstants.java

```diff
@@ -13,7 +13,7 @@
         public static final String ID = "ID";
         public static final String TENANT_ID = "TENANT_ID";
         public static final String ORIGIN = "ORIGIN";
-        public static final String UUID = "UUID";
+        public static final String UNIQUE_ID = "UUID";
```

Comment:
If we using UUID as the name then while importing it is conflicting with java.utils.UUID that's why changed the name so that static imports will work for java UUID as well.